stretch3_ws/src/isaacsim/scripts/run_isaacsim.py

```
@@ -0,0 +1,189 @@
+            else:
+                os.environ["LD_LIBRARY_PATH"] = f"{os.getenv('LD_LIBRARY_PATH')}:{filepath_root}/exts/isaacsim.ros2.bridge/{args.ros_distro}/lib"
+                specific_path_to_remove = f"/opt/ros/{args.ros_distro}"
+                version_to_remove = "jazzy" if args.ros_distro == "humble" else "humble"
+                update_env_vars(version_to_remove, specific_path_to_remove, "LD_LIBRARY_PATH")
+                update_env_vars(version_to_remove, specific_path_to_remove, "PYTHONPATH")
+                update_env_vars(version_to_remove, specific_path_to_remove, "PATH")
+        
+        elif args.ros_installation_path:
+            # If a custom ros installation path is provided
+            if sys.platform == "win32":
+                proc = subprocess.Popen(f"call {args.ros_installation_path}", shell=True, start_new_session=True)
```

Ruff (0.14.10)

`subprocess` call with `shell=True` identified, security issue

(S602)